apps/ledger-live-mobile/src/components/Web3AppWebview/WalletAPIWebview.tsx

```diff
@@ -93,31 +98,33 @@ WalletAPIWebview.displayName = "WalletAPIWebview";
 
 function renderLoading() {
   return (
-    <View style={styles.center}>
+    <View style={styles().center}>
       <ActivityIndicator size="small" />
     </View>
   );
 }
 
-const styles = StyleSheet.create({
-  root: {
-    flex: 1,
-  },
-  center: {
-    flex: 1,
-    flexDirection: "column",
-    alignItems: "center",
-    position: "absolute",
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    justifyContent: "center",
-  },
-  webview: {
-    flex: 1,
-    width: "100%",
-    height: "100%",
-    backgroundColor: "transparent",
-  },
-});
+const styles = (error = false) =>
+  StyleSheet.create({
+    root: {
+      flex: 1,
+    },
+    center: {
+      flex: 1,
+      flexDirection: "column",
+      alignItems: "center",
+      position: "absolute",
+      top: 0,
+      left: 0,
+      right: 0,
+      bottom: 0,
+      justifyContent: "center",
+    },
+    webview: {
+      display: error ? "none" : "flex",
+      flex: 1,
+      width: "100%",
+      height: "100%",
+      backgroundColor: "transparent",
+    },
+  });
```

Comment:
We should keep the non conditional types in the `StyleSheet.create` with no wrapping function to avoid recreating the styles on every renders

IMO we should use `style={[styles.webview, { display: error ? "none" : "flex" }]}`

Reply:
oooh react native style can take an array.. cool!